src/create-glamorous.js

```diff
@@ -78,7 +78,10 @@ export default function createGlamorous(splitProps) {
         }
 
         render() {
-          const props = this.props
+          const {children, ...props} = this.props
+          const spreadableChildren = Array.isArray(children)
```

Comment:
Is there a `React.Children` helper here that could alleviate this a bit? Maybe `React.Children.toArray`?